src/api/lib/tasks/dev.rake

```diff
@@ -170,6 +170,57 @@ namespace :dev do
     end
   end
 
+  namespace :notifications do
+    task data: :environment do
+      unless Rails.env.development?
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      requestor = User.where(login: 'Requestor').first || create(:confirmed_user, login: 'Requestor')
+      admin = User.where(login: 'Admin').first
+      home_admin = admin.home_project
+
+      unless home_admin
+        home_admin = create(:project, name: admin.home_project_name)
+        create(:relationship, project: home_admin, user: admin, role: Role.hashed['maintainer'])
+      end
+
+      branched_project = Project.where(name: requestor.branch_project_name('home:Admin')).first ||
+                         create(:project, name: requestor.branch_project_name('home:Admin'))
+      admin_package = Package.where(name: 'admin_package').first ||
+                      create(:package_with_file, name: 'admin_package', project: home_admin, file_content: 'from admin home')
```

Comment:
@saraycp how about adding a timestamp or a random number to the package name? this way we could run the task a few times if a few more fake notifications are needed, without having always the same package name :)

Reply:
Good idea!

Reply:
@krauselukas I folloed your suggestion and added different names. But also improved to make it easier to generate many notification at a time.

Run this task with `rails "dev:notifications:data[3]"` replacing 3 with any number to indicate how many times you want this code to be executed.